QueueProcessors/AutoreductionProcessor/autoreduction_processor.py

```diff
@@ -12,7 +12,8 @@
 from QueueProcessors.AutoreductionProcessor.autoreduction_logging_setup import logger
 # pylint:disable=no-name-in-module,import-error
 from QueueProcessors.AutoreductionProcessor.settings import MISC
-from settings import ACTIVEMQ
+# pylint:disable=relative-import
+from utils.settings import ACTIVEMQ
```

Comment:
This should not be in this PR

Reply:
This change is required in order for the build to pass.

Reply:
I'm confused as to how this actually works. utils.settings doesn't have anything called `ACTIVEMQ` anymore it is now `ACTIVEMQ_SETTINGS`. I'll take a look at resolving this